.eslintrc.js

```diff
@@ -0,0 +1,10 @@
+module.exports = {
+  root: true,
+  // This tells ESLint to load the config from the package `eslint-config-grafbase`
```

Comment:
is this comment needed when this is expected behavior?

Reply:
Removed